$NetBSD: patch-is,v 1.1 2008/01/05 15:15:02 rillig Exp $

isis_spf.c:530: error: invalid storage class for function 'print_route'

--- src/isis_spf.c.orig	1999-02-18 00:30:52.000000000 +0100
+++ src/isis_spf.c	2008-01-05 16:12:36.000000000 +0100
@@ -518,6 +518,8 @@ get_graph_entry __PF3(sysid, char *,
  * the pending database.  This routine only supports the 
  * default metric for now.
  */ 
+static void print_route __PF3(gpt, GRAPH_PT, logPri, int, printOld, int);
+
 static void
 spf_check_pending __PF6(dbEntry, GRAPH_PT, 		/* the route we had for dest (could be freshly initialized) */
 		  	internal_cost, int,		/* internal cost for the new route */
@@ -527,7 +529,6 @@ spf_check_pending __PF6(dbEntry, GRAPH_P
 							/* neighbors, added by init_paths) */
 		  	asp, as_path *)
 {
-	static void print_route();
 	int destType = dbEntry->g.gtype;
 
 #define	UseNewRoute \
